cmd/zed/zed.d/zed.rc

```diff
@@ -54,7 +54,7 @@
 # Send notifications for 'ereport.fs.zfs.data' events.
 # Disabled by default
 #
-#ZED_NOTIFY_DATA=1
+#ZED_NOTIFY_DATA=0
```

Comment:
It looks like the "default" here is more the empty string than `0`. `cmd/zed/zed.d/data-notify.sh` has: `[ -n "${ZED_NOTIFY_DATA}" ] || exit 3`. So it looks like `0` would have the same effect as `1`.

Reply:
interesting. good catch. I saw elsewhere it checked for 1.

Reply:
I think you have this backwards. ZED_NOTIFY_DATA is the one that should be the empty string. Arguably ZED_SCRUB_AFTER_RESILVER could be the empty string too, but 0 is okay for that one.